tests/fixtures/marc/marc_record_all_fields.xml

```diff
@@ -18,6 +18,22 @@
         <datafield tag="082" ind1="0" ind2=" ">
             <subfield code="a">Dewey Call Number 3</subfield>
         </datafield>
+        <datafield tag="100" ind1="1" ind2=" ">
+            <subfield code="a">Tran, Phong,</subfield>
+            <subfield code="e">composer,</subfield>
+            <subfield code="e">performer.</subfield>
+        </datafield>
+        <datafield tag="110" ind1="2" ind2=" ">
+            <subfield code="a">Ceeys (Musical group),</subfield>
+            <subfield code="e">composer,</subfield>
+            <subfield code="e">performer.</subfield>
+        </datafield>
+        <datafield tag="111" ind1="2" ind2=" ">
```

Comment:
I’ve included a full 111 field from our local data, these (and 711s) seem to be formatted this way in almost all the examples I found so I’m not sure if we want to process this differently than the Go version. The punctuation makes it quirky